astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,38 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, suppress_error=True):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
+        suppress_error : bool (default value is False)
```

Comment:
I don't know if we should provide the `(default value is False)` here. I think that belongs into the description.